src/op.rs

```diff
@@ -771,6 +787,16 @@ impl<'input, Endian> Operation<'input, Endian>
                 Ok((newbytes, Operation::ImplicitValue { data: data.into() }))
             }
             constants::DW_OP_stack_value => Ok((bytes, Operation::StackValue)),
+            constants::DW_OP_implicit_pointer |
+            constants::DW_OP_GNU_implicit_pointer => {
+                let (newbytes, value) = try!(parse_offset(bytes, format));
+                let (newbytes, byte_offset) = try!(parse_signed_lebe(newbytes));
+                Ok((newbytes,
+                    Operation::ImplicitPointer {
+                    value: DebugInfoOffset(value),
```

Comment:
The indentation looks off here but I can't tell if that is github or just needing rustfmt.